.gitignore

```diff
@@ -35,3 +35,6 @@ src/applications/bmqbrkr/etc/etc
 
 # 'sim_cpp11_features.pl' backups
 *.bak
+
```

Comment:
This may not be the best to commit to the main repo, since it's for your personal usage.  [Git looks in a few place for files to ignore](https://git-scm.com/docs/gitignore): `.gitignore` is one, and is meant to be committed and distributed to everyone; `.git/info/exclude` is another, and only applies to your local repo.